akka-actor/src/main/scala/akka/serialization/Serialization.scala

```diff
@@ -1,5 +1,6 @@
 /**
  * Copyright (C) 2009-2018 Lightbend Inc. <https://www.lightbend.com>
+ * Copyright (c) 2018      Gael Breard, Orange: issue #24321 add a method for transport information in public API
```

Comment:
@gbrd Is this important for you? We normally don't add such copyright notes in contributions. 

If we copy code from an external library where the license permits that we keep copyright attribution but that is a different scenario.

Reply:
It is asked by my compagny.

Reply:
@gbrd We have asked for advice from our legal experts.

The conclusion is that you are making contributions per the CLA and can claim that those are made by the company (or the individual), but we don't include that information in the license that accompanies the software.  The Github pull request is the 'tracking' mechanism that shows who made a contribution.

If you are not fine with that decision I'm afraid we have to close this PR.

Reply:
ok go without it (I guess tracking via PR is enough for my company, if it's the rule for yours)

Reply:
thanks for understanding